saleor/graphql/webhook/subscription_types.py

```diff
@@ -1161,12 +1162,33 @@ def resolve_order(root, info: ResolveInfo):
 
 
 class FulfillmentCreated(SubscriptionObjectType, FulfillmentBase):
+    notify_customer = graphene.Boolean(
+        description="If true, send a notification to the customer." + ADDED_IN_316,
```

Comment:
Shouldn't be: `If true, the app should send a notification to the customer.` ? IMHO Now is little bit not clear who should send a notification 😅

Reply:
I believe English is not my best skill 🙈